pkg/k8s/apis/cilium.io/client/crds/v2alpha1/ciliumbgppeeringpolicies.yaml

```diff
@@ -220,8 +220,12 @@ spec:
                       type: array
                     serviceSelector:
                       description: "ServiceSelector selects a group of load balancer
-                        services which this virtual router will announce. \n If empty
-                        / nil no services will be announced."
+                        services which this virtual router will announce. The loadBalancerClass
+                        for a service must be nil or specify a class supported by
+                        Cilium, e.g. \"io.cilium/bgp-control-plane\". Refer to the
+                        following document for additional details regarding load balancer
+                        classes: \n   https://kubernetes.io/docs/concepts/services-networking/service/#load-balancer-class
```

Comment:
Are these newlines supported?

Reply:
The newlines are not reflected in `kubectl explain`. However, existing fields have the same issue and the newlines improve readability.